### PR TITLE
BETA: Register danix.is-a.dev

### PR DESCRIPTION
**Pull request by ghost**
Added `danix.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by danix421** [comment]
already used is-a.dev in the past (before the new dashboard was made), though I removed my domain a few months back since I wasn't using it a lot.

File: domains/danix.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "danix421",
+        "email": "danix@disroot.org"
+    },
+    "record": {
+        "CNAME": "danix421.github.io"
```
Comment (DEV-DIBSTER) [inline domains/danix.json:7]: 404 CNAME record.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/danix.json (hunk 1 above)
Comment (danix421) [reply domains/danix.json:7]: it's meant to be like that. I haven't commited the page itself yet, since I wanted to work around the domain first.

Comment (WilliamDavidHarrison) [reply domains/danix.json:7]: @danix421 We do not support domain reservation, so you will need to at least put something there like maybe some basic contact details and info before you can get the domain.

**Pull request closed without merging**